HISTORY.rst

```diff
@@ -0,0 +1,8 @@
+=======
+History
+=======
+
+0.1.0 (2018-03-08)
```

Comment:
Is this premature? I do not see such a version published on https://pypi.python.org/pypi